---
layout: post
title: NeurIPS 2023 Health & Bio Conference Review
subtitle: What happened in health and bio at NeurIPS 2023?
howlong: 5 min
comments: false
author: Patrick Schwab
image: http://schwabpatrick.com/img/neurips23-img.png
twitter: https://twitter.com/schwabpa/status/1599423696048832512?s=20
tags:
- conference review
---
Unable to keep up with the deluge of amazing work happening in ML for Biology and Health at NeurIPS this year?

We've got you covered with a concise summary of NeurIPS 2023 content focussed at the exciting intersection of Biology, Health and AI!

<nav id="toc"><h3><a data-toggle="collapse" href="#toccontent" role="button" aria-expanded="false" aria-controls="toccontent"><i class="fa fa-bars"></i>  Table of contents</a></h3></nav>

<h3>Keynotes</h3>

<h4>Delusion of Scaling and Democratization of Generative Models - Björn Ommer</h4>

Björn Ommer (Stable Diff) starts us off with defining human vision as grasping things without touch and perception as a process of prediction.
He argues intelligence is learning under finite resources to support research outside scaling & makes case for accessible and open models.

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/om1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/om2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/om3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/om4.png"></galleryitem> 
</div>

<h4>Systems for Foundation Models, and Foundation Models for Systems - Chris Re</h4>

In a captivating talk, Chris Ré shows us the potential of foundation models (FMs) for systems and introduces the paradigm shift behind FMs: from solve in detail, to solve in general
He outlines data cleaning as a valuable example where FM have made huge strides (also OSS models)

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/om1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/om2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/om3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/om4.png"></galleryitem> 
</div>

<h3>Selected papers</h3>

<h4>De novo Drug Design using Reinforcement Learning with Multiple GPT Agents</h4>

Xinyuan Hu et al (Microsoft Research, Tsinghua University) introduce a multi GPT agent framework for de-novo drug design with RL with the goal of promoting diverse candidate generation.
Agents use memory, and they show candidate inhibitors for SARS-CoV2.

<a href="https://openreview.net/pdf?id=1B6YKnHYBb">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/de1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/de2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/de3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/de4.png"></galleryitem> 
</div>

<h4>Implicit Transfer Operator Learning: Multiple Time-Resolution Models for Molecular Dynamics</h4>

M Schreiner et al (Chalmers University, DTU) address the key problem of reconciling small time steps and long convergence times in molecular dynamics (MD) simulations. 

Their new multi-timestep transfer operator shows self-consistent stochastic dynamics across time-scales.

<a href="https://openreview.net/forum?id=1kZx7JiuA2">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/md1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/md2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/md3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/md4.png"></galleryitem> 
</div>

<h4>xTrimoGene: An Efficient and Scalable Representation Learner for Single-Cell RNA-Seq Data</h4>

J Gong et al (BioMap & Tsinghua University) present xTrimoGene - a scalable representation learner for scRNAseq data.

Using self-supervised learning and an encoder-decoder architecture, they show respectable performance on cell type annotation, response prediction and drug combination prediction tasks.

<a href="https://www.biorxiv.org/content/10.1101/2023.03.24.534055v1">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/xt1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/xt2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/xt3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/xt4.png"></galleryitem> 
</div>

<h4>HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution</h4>

Eric Nguyen et al (Stanford) present HyenaDNA a long-range, efficient single-nucleotide model with sequence windows up to an impressive 1m base pairs that is trained on self-supervised next nucleotide prediction task.
They show scale benefits performance on species classification.

<a href="https://arxiv.org/abs/2306.15794">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/hd1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/hd2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/hd3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/hd4.png"></galleryitem> 
</div>


<h4>Protein Design with Guided Discrete Diffusion</h4>

N Gruver et al (Prescient Design) present a discrete guided diffusion approach to antibody (Ab) design that leverages multi-objective optimisation to optimize Ab properties. 

They show impressive improvements in iterative experimental rounds in binding and expression of Ab candidates.

<a href="https://arxiv.org/abs/2305.20009">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/pd1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/pd2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/pd3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/pd4.png"></galleryitem> 
</div>


<h4>Knowledge-based in silico models and dataset for the comparative evaluation of mammography AI for a range of breast characteristics, lesion conspicuities and doses</h4>

Elena Sizikova et al (US FDA) introduce an approach for synthetic data generation to evaluate AI mammography algorithms and address limitations of evaluating against real-world data, incl biases and small dataset sizes
They use their approach to evaluate AI models for mammography.

<a href="https://arxiv.org/abs/2310.18494">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/mg1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/mg2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/mg3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/mg4.png"></galleryitem> 
</div>


<h4>FABind: Fast and Accurate Protein-Ligand Binding</h4>

Qizhi Pei et al (Microsoft Research & Renmin University) introduce a fast and accurate method (FABind) for protein-ligand binding prediction without pocket information. 
Their approach combines pocket prediction and docking and achieves SOTA quantitative results on PDBbind data.

<a href="https://arxiv.org/abs/2310.06763">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/fa1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/fa2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/fa3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/fa4.png"></galleryitem> 
</div>

<h4>SG×P : A Sorghum Genotype × Phenotype Prediction Dataset and Benchmark</h4>

As is the case in human health, understanding the relationship between genetic background and observed outcomes (phenotypes) is of paramount importance also in plant sciences.

Z Zhang et al (George Washington University) create a Sorghum dataset (500k+ images) for studying these relationships.

<a href="https://github.com/SLUVisLab/sorghum_snp_prediction">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ps1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ps2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ps3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ps4.png"></galleryitem> 
</div>

<h4>AbDiffuser: Full-Atom Generation of In-Vitro Functioning Antibodies</h4>

K Martinkus et al (Prescient Design) present AbDiffuser - a generative approach to antibody design that improves protein diffusion by integrating domain knowledge and physics-based constraints

They present exciting evidence that some of their candidates were novel HER2 binders

<a href="https://arxiv.org/abs/2308.05027">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ab1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ab2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ab3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ab4.png"></galleryitem> 
</div>

<h4>ProteinGym: Large-Scale Benchmarks for Protein Fitness Prediction and Design</h4>

In recognition that proteins need to stay fit too, P Notin et al (Oxford University & Harvard University) introduce ProteinGym - a benchmark for protein fitness prediction that features  curated deep mutational scanning & clinical datasets, relevant baseline models and metrics.

<a href="https://www.biorxiv.org/content/10.1101/2023.12.07.570727v1">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/pg1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/pg2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/pg3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/pg4.png"></galleryitem> 
</div>

<h4>RaLEs: a Benchmark for Radiology Language Evaluations</h4>

J Z Chaves et al (Stanford) address the challenge of evaluating natural language models for radiological findings. 

They introduce a benchmark and find that advances in more general domains do not necessarily translate to Radiology and highlight opportunities for future work

<a href="https://openreview.net/pdf/f1783ddf0a1c8990511ede94c4ebd18226d4280c.pdf">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ra1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ra2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ra3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/ra4.png"></galleryitem> 
</div>

<h4>Disentangled Wasserstein Autoencoder for T-Cell Receptor Engineering</h4>

Tianxiao Li et al (@Yale) present an exciting advance in engineering T-cell receptors that balances maintaining the overall structure with modifying the functional site. They use a disentangled Wasserstein autoencoder that demonstrates quality & quantity of results in experiments.

<a href="https://openreview.net/pdf/f1783ddf0a1c8990511ede94c4ebd18226d4280c.pdf">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/neurips2023/tc1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/tc2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/tc3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/neurips2023/tc4.png"></galleryitem> 
</div>

<h3>Conclusion</h3>

Overall, exciting observations are:
- the emerging synthesis of deep learning and causality
- the ongoing reinvention of the entire healthcare stack with a data-driven lens
- increasing bio/health presence, but (in contrast to ICML) still small vs vision and text at #NeurIPS2022

<br/>

DISCLAIMER: The above list is a personal curation that most certainly missed many key contributions (in particular the many excellent workshop & competition contributions!) and is only intended to be a starting point for your own exploration.