---
layout: post
title: ICLR/MLDD 2024 Health & Bio Conference Review
subtitle: What happened in health and bio at ICLR 2023?
howlong: 5 min
comments: false
author: Patrick Schwab
image: http://schwabpatrick.com/img/iclr2024-img.jpeg
twitter: https://twitter.com/schwabpa/status/1736113182215672316
tags:
- conference review
---
So much happening in ML for Biology and Health at ICLR and MLDD this year and no time to catch up?

As always, we've got you covered with a concise summary of ICLR and MLDD 2024 content focussed at the buzzing intersection of Bio, Health and AI!

<nav id="toc"><h3><a data-toggle="collapse" href="#toccontent" role="button" aria-expanded="false" aria-controls="toccontent"><i class="fa fa-bars"></i>  Table of contents</a></h3></nav>

<h3>Machine Learning for Drug Discovery (MLDD)</h3>

The first event of the week was the Machine Learning for Drug Discovery (MLDD) symposium featuring a stellar lineup of speakers: Lucy Colwell, David Baker, Pascal Notin, Thore Graepel, Smita Krishnaswamy, Shantanu Singh, and Žiga Avsec.

<h4>Generating and Analyzing Molecules via Learnable Geometric Scattering - Smita Krishnaswamy</h4>

Smita Krishnaswamy (Yale University) introduced graph scattering synthesis (GRASSY) that permits the steered generation of molecules via latent space interpolation.

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/smita11.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/smita22.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/smita33.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/smita44.png"></galleryitem> 
</div>

<h4>Reflections on AI, Rejuvenation and Emergence - Thore Graepel</h4>

Thore Graepel (Altos Labs) presented an insightful view on the intriguing parallels between the scientific fields of rejuvenation and AI and between agent-based reinforcement learning and drug discovery with a view on emergence across ML systems and medicine.

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/tg1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/tg2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/tg3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/tg4.png"></galleryitem> 
</div>

<h4>Accurate proteome-wide missense variant effect prediction with AlphaMissense - Žiga Avsec</h4>

Žiga Avsec (Google DeepMind) introduced us to the exciting world of genetic sequence models with their recent work on accurately predicting the pathogenicity of missense variants using AlphaMissense.

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/za1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/za2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/za3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/za4.png"></galleryitem> 
</div>

<h4>Cell Painting powers next-generation phenotypic drug discovery - Shantanu Singh</h4>

Shantanu Singh (Broad Institute of MIT and Harvard) gave us a whirlwind tour through the exciting world of using cell painting datasets for drug discovery with many applications from gene characterisation to measuring complex phenotypes to virtual screening.

<a href="https://broad.io/CarpenterSinghSlides">[Slides link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/ss1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/ss2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/ss3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/ss4.png"></galleryitem> 
</div>

<h4>Machine Learning to predict protein function from sequence with therapeutic applications - Lucy Colwell</h4>

Lucy Colwell (Google Research and University of Cambridge) showcased outstanding work on (1) better understanding the sequence to function link and using this understanding to design candidate sequences  and on (2) combining language and sequences to find proteins via text descriptions of their function.

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/lc1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/lc2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/lc3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/lc4.png"></galleryitem> 
</div>

<h4>Hybrid protein language models for fitness prediction and design - Pascal Notin</h4>

Pascal Notin (Harvard University) introduced the audience to the state-of-the-art in protein language models with applications to human variant annotation, viral escape prediction and the using conditional generation capabilities of pLMs to design proteins with specific properties.

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/pn1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/pn2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/pn3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/pn4.png"></galleryitem> 
</div>

<h4>De novo protein design - David Baker</h4>

David Baker (University of Washington) showcased their remarkable toolkit for protein structure prediction and design that is able to generate proteins binding to receptors, nanobodies binding to hotspots, form symmetric assemblies and that bind DNA with immense potential for medicine.

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/db3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/db4.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/db5.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/db6.png"></galleryitem> 
</div>

<h3>Keynotes</h3>

<h4>Machine Learning in Prescient Design's Lab-in-the-Loop Antibody Design - Kyunghyun Cho</h4>

In a highly recommended keynote, Kyunghyun Cho (Prescient Design/Genentech) kicks us off with a tour of their lab-in-the-loop for antibodies - showcasing challenges (large design space, multiple objectives), methods (Walk Jump Sampling), performance and the big picture in drug discovery.

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/kc1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/kc2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/kc3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/kc4.png"></galleryitem> 
</div>

<h4>The emerging science of benchmarks - Moritz Hardt</h4>

Moritz Hardt (Prescient Design/Genentech) starts us off with (...)

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/mh1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/mh2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/mh3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/mh4.png"></galleryitem> 
</div>

<h3>Selected papers</h3>

<h4>De novo Drug Design using Reinforcement Learning with Multiple GPT Agents</h4>

Xinyuan Hu et al (Microsoft Research, Tsinghua University) introduce a multi GPT agent framework for de-novo drug design with RL with the goal of promoting diverse candidate generation.
Agents use memory, and they show candidate inhibitors for SARS-CoV2.

<a href="https://openreview.net/pdf?id=1B6YKnHYBb">[Paper link]</a>

<div class="gallery">
<galleryitem src="http://schwabpatrick.com/img/iclr2024/pn1.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/pn2.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/pn3.png"></galleryitem> 
<galleryitem src="http://schwabpatrick.com/img/iclr2024/pn4.png"></galleryitem> 
</div>

<h3>Conclusion</h3>

Overall, key observations are:
- Health and bio is coming of age in the ML community (though still mostly relegated to posters)
- Sequence learning, proteomics and design tasks are emerging as areas of especially high activity and promise for ML in bio

<br/>

DISCLAIMER: The above list is a personal curation that most certainly missed many key contributions (in particular the many excellent workshop & competition contributions!) and is only intended to be a starting point for your own exploration.